src/view.js

```diff
@@ -136,12 +135,6 @@ const View = Backbone.View.extend({
   _renderTemplate() {
     const template = this.getTemplate();
 
-    // Allow template-less views
-    if (template === false) {
-      deprecate('template:false is deprecated.  Use _.noop.');
-      return;
-    }
-
     // Add in entity data and template context
     const data = this.mixinTemplateContext(this.serializeData());
```

Comment:
having this line still execute is the odd part in the code.
the template context and serializeData will do work that might not get used.
I can't figure out what to do about that needless work, but template false did solve it.

Reply:
I do think that there will be some less than ideal scenarios for when users are wanting views that do not contain renderable content.  Granted I would also wonder what the use case is for `templateContext` and to some degree model data when no rendering is to be done..  So while, yes, there's some extra work being done, I don't see the use case where it is _really_ making that much of a difference.

On the contrary `template: false` has been predominately used as a method for attaching children within `onRender` when the view itself _should not be rendered_  That is a terrible terrible use of this API and we should do whatever we can to avoid it being the prominent case.  Instead I believe we should provide a better hook for when child views should be added as `onRender` is not good enough.

So I would argue that keeping this functionality is far worse than removing.

Reply:
In light of https://github.com/marionettejs/backbone.marionette/issues/3484 can we push this through @rafde ?